dumper/app/src/main/resources/oracle-stats/dba/native/app-schemas-pdbs.sql

```diff
@@ -13,20 +13,44 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 SELECT
-  NULL "Source",
-  NULL "DbId",
-  NULL "PdbId",
-  NULL "PdbName",
-  NULL "Status",
-  NULL "Logging",
-  NULL "ConId",
-  NULL "ConUid",
+  D.source "Source",
```

Comment:
I would recommend to skip sqls related to `pdbs` for single tenant instance. This query should work for the single tenant instance with version 12c+ but it will fail on Oracle 11c due to missing dba_pdbs table.
This can addressed later. For now the highest priority is to support Oracle 12c+. We can work on version 11 later.

Reply:
I agree, this file was just a random example so that https://github.com/google/dwh-migration-tools/pull/482 would have something to run. As discussed, the current priority is to first move the required, native sqls and the next group will be the sqls using AWR.